src/pages/homeCollection/kitAssembly.js

```diff
@@ -125,50 +125,49 @@ const processAssembledKit = () => {
 
-      if (queryScannedBarcodeValue !== confirmScannedBarcodeValue) {
-        triggerErrorModal('Return tracking number doesn\'t match.');
-        return;
-      }
-      else if (scannedBarcodeValue.length === 0 || supplyKitIdValue.length === 0 ||  returnKitIdValue.length === 0 ||
-        collectionCupIdValue.length === 0 || collectionCardIdValue.length === 0 || document.getElementById('dropdownSites').innerHTML !== 'Mouthwash') {
-          triggerErrorModal('One or more fields are missing.');
-          return
-        }
-      else {
-        kitObj[conceptIds.returnKitTrackingNum] = scannedBarcodeValue;
-        kitObj[conceptIds.supplyKitId] = supplyKitIdValue;
-        kitObj[conceptIds.returnKitId] = returnKitIdValue;
-        kitObj[conceptIds.collectionCupId] = collectionCupIdValue
-        kitObj[conceptIds.collectionCardId] = collectionCardIdValue;
-        kitObj[conceptIds.kitType] = conceptIds.mouthwashKitType;
-        const responseStoredStatus = await storeAssembledKit(kitObj);
-        if (responseStoredStatus) {
-          document.getElementById('scannedBarcode').value = ``;
-          document.getElementById('supplyKitId').value = ``;
-          document.getElementById('returnKitId').value = ``;
-          document.getElementById('cupId').value = ``;
-          document.getElementById('cardId').value = ``;
-          document.getElementById("showMsg").innerHTML = ``;
+        if (queryScannedBarcodeValue !== confirmScannedBarcodeValue) {
+            triggerErrorModal('Return Kit tracking number doesn\'t match.');
+            return;
+        } else if (scannedBarcodeValue.length === 0 || supplyKitIdValue.length === 0 ||  returnKitIdValue.length === 0 ||
+            collectionCupIdValue.length === 0 || collectionCardIdValue.length === 0 || document.getElementById('dropdownSites').innerHTML !== 'Mouthwash') {
+            triggerErrorModal('One or more fields are missing.');
+            return
+        } else {
+            kitObj[conceptIds.returnKitTrackingNum] = scannedBarcodeValue;
+            kitObj[conceptIds.supplyKitId] = supplyKitIdValue;
+            kitObj[conceptIds.returnKitId] = returnKitIdValue;
+            kitObj[conceptIds.collectionCupId] = collectionCupIdValue
+            kitObj[conceptIds.collectionCardId] = collectionCardIdValue;
+            kitObj[conceptIds.kitType] = conceptIds.mouthwashKitType;
+            const responseStoredStatus = await storeAssembledKit(kitObj);
+            if (responseStoredStatus) {
+            document.getElementById('scannedBarcode').value = ``;
+            document.getElementById('scannedBarcode2').value = ``;
+            document.getElementById('supplyKitId').value = ``;
+            document.getElementById('returnKitId').value = ``;
+            document.getElementById('cupId').value = ``;
+            document.getElementById('cardId').value = ``;
+            document.getElementById("showMsg").innerHTML = ``;
+            }
```

Comment:
Is there time to wrap this in a try/catch or handle the else case if `responseStoredStatus` doesn't return in the expected format? Maybe with `triggerErrorModal` in the catch block? I haven't looked into that function, so mabye it's already handled.

Also, indentation after the if statement.

Reply:
Thanks for the comments. I'll make the changes.